linux-system-roles.spec

```diff
@@ -62,13 +62,17 @@ Patch5: rhel-system-roles-%{rolename5}-prefix.diff
 # Not suitable for upstream, since the files need to be executable there
 Patch52: network-permissions.diff
 
+Patch11: rhel-system-roles-postfix-pr5.diff
```

Comment:
@pcahyna We need this file in git, and you need a comment about why it's important (it is obvious) and that it was merged.

https://github.com/linux-system-roles/postfix/commit/91c29098e0a7c60dafd74dfee8d24bc520a9f0b3.patch

This is my only comment. If you work it off & do a test build, you can merge it.

Reply:
Actually it is better to point to the upstream release which has it already and drop it.